projects/observability/src/pages/apis/service-detail/instrumentation/components/org-score.component.ts

```diff
@@ -0,0 +1,31 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ServiceInstrumentationService } from '../service-instrumentation.service';
+
+@Component({
+  styleUrls: ['./org-score.component.scss'],
+  selector: 'ht-service-instrumentation-org-score',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="service-instrumentation-org-score">
+      <div>
```

Comment:
✅ Yes. It's kinda redundant right now but will need to add it back when we add BU score since we need container divs for both. The extra `div` is redundant right now but the elements are getting misaligned without it for some reason.

<img width="306" alt="Screenshot 2022-09-13 at 4 07 30 PM" src="https://user-images.githubusercontent.com/29686866/189880729-e33e9c51-df9f-4384-86fd-b63ab3c2f476.png">

Reply:
Found the problem. I was using `place-items`, which centers the elements both horizontally and vertically. The horizontal centering was making the heading go off. Replaced `place-items` with `align-items`.

We still need the extra div though. Removing it makes the container treat both the elements as separate units and space them evenly vertically. This adds too much vertical gap between the heading and score.

<img width="306" alt="Screenshot 2022-09-13 at 7 36 45 PM" src="https://user-images.githubusercontent.com/29686866/189924113-d59cc23f-7d6e-4e35-a52d-6854889e1e3d.png">

What we want is for the container to treat both divs as a single unit meant to be centered vertically.

<img width="306" alt="Screenshot 2022-09-13 at 7 39 15 PM" src="https://user-images.githubusercontent.com/29686866/189924338-22e17fd5-dce9-4d7c-b27c-011ca1ca9bb8.png">

This would help us add the BU score later without any additional CSS change too, since the container will then see 2 divs to space out evenly vertically instead of 4.

<img width="233" alt="Screenshot 2022-09-13 at 7 43 43 PM" src="https://user-images.githubusercontent.com/29686866/189924836-f1fa7fbb-fd2d-44f5-bf51-5c9b8c976c77.png">